lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -149,6 +177,9 @@ def get(self, request, *args, **kwargs):
             'course_tools': course_tools,
             'dates_widget': dates_widget,
             'handouts_html': handouts_html,
+            'has_visited_course': has_visited_course,
+            'resume_course_title': resume_course_title,
+            'resume_course_url': resume_course_url,
```

Comment:
Would it make sense to have like a `resume_button` grouping or something here? I don't super care. Just noticed that the other pieces of this seem grouped.

Reply:
Yeah at first I was hesitant because of the `resume_course_title` field, but if we're getting rid of that I think it makes sense to group these!